### PR TITLE
Fixes to php-cs-fixer: ignore cache file in Git, fix Makefile usage & apply to…

### PR DESCRIPTION
**Pull request by jmsche**
Hi,

This PR aims to fix the following:

- includes .php-cs-fixer.php to the files to fix (& apply fixes to it)
- ignore the Jane generated files in the tests directory
- ignores php-cs-fixer cache file in Git
- fix Makefile usage to make it consistent with CI & config

### CODE REVIEW

**Comment by damienalexandre** [comment]
Perfect, thank you :+1:

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 /tests/Jane/generated/*
 !/tests/Jane/generated/.gitkeep
 .php_cs.cache
```
Comment (jdreesen) [inline .gitignore:6]: This should not be needed anymore, as it's the name of the cache file from PHP-CS-Fixer 2 (see: [upgrade docs](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#rename-of-files))

Comment (lyrixx) [reply .gitignore:6]: Would you mind sending a PR? :)

Comment (jdreesen) [reply .gitignore:6]: Here it is: #85

**Pull request merged**